plugin/efficientNMSPlugin/README.md

```diff
@@ -134,6 +135,10 @@ The algorithm is highly sensitive to the selected `score_threshold` parameter. W
 
 Depending on network configuration, it is usually more efficient to provide raw scores (pre-sigmoid) to the NMS plugin scores input, and enable the `score_activation` parameter. Doing so applies a sigmoid activation only to the last `max_output_boxes` selected scores, instead of all the predicted scores, largely reducing the computational cost.
 
+#### Class Independent NMS
+
+Some object detection networks/architectures like YOLO series need to use class independent nms operations. Do class independent nms if `class_agnostic` is enabled, otherwise, different class would do nms respectively.
```

Comment:
```suggestion
Some object detection networks/architectures like YOLO series need to use class-independent NMS operations. If `class_agnostic` is enabled, class-independent NMS is performed; otherwise, different classes would do NMS separately.
```